feature-libs/organization/administration/components/unit/links/children/unit-children.component.ts

```diff
@@ -22,9 +22,5 @@ export class UnitChildrenComponent {
     ? this.currentUnitService.item$
     : of({ active: true });
 
-  /**
-   * @deprecated since 3.0.10
-   * Include CurrentUnitService in constructor for bugfix #10688.
-   */
-  constructor(protected currentUnitService?: CurrentUnitService) {}
+  constructor(protected currentUnitService: CurrentUnitService) {}
```

Comment:
Still to do migrations for constructors?

Reply:
Already done by @Zeyber few months ago 😁